runbotics/runbotics-orchestrator-ui/src/src-app/store/StorageManager.service.ts

```diff
@@ -0,0 +1,62 @@
+export enum StorageKey {
+    DESTINATION_PAGE='destination_page',
+    ACCESS_TOKEN='access_token'
+}
+
+class StorageManagerService {
+    // --= Private =--
+    private static setItem(key: string, value: string): void {
+        localStorage.setItem(key, value);
+    }
+
+    private static readItem(key: string): string {
+        try {
+            const data = localStorage.getItem(key);
+            return data;
+        } catch(error) {
+            // eslint-disable-next-line no-console
+            console.log(error);
```

Comment:
the final solution should not contain console logs -> remove it

Reply:
tbh errors should be handled with it's usage because we will lose this information here. In components use can display a toast or do smthing different